templates/react/utils/itemToLinks.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import {Link} from 'react-router-dom';
+import {API_PATH} from '../api/_entrypoint';
+
+export default function (itemArray) {
+  if (Array.isArray(itemArray)) {
+    return (
```

Comment:
I like the one liner :) change it already and will commit.
About minimizing number of files not sure. 
I was thinking to get all action types and create const values in separate file instead using strings.
WDYT @dunglas ?